### ISSUE TITLE
Feature: Segments

### ISSUE THREAD

**Issue by tkell** [algorithmic-music-exploration/amen#83]
As per @Cortexelus on the timbre thread, it would be nice to have Echo Nest style segments as a Timing.  

Will doing this make anything weird, in terms of how we handle Feature data with pandas?  I don't think so.

**Comment by bmcfee** [comment]
Nope, it's just another type of time slicer.

There is an open issue in librosa to pull in some of Tristan's tricks for segment/onset timing: https://github.com/librosa/librosa/issues/305 ; probably best to wait on that to be merged before going forward.

**Comment by tkell** [comment]
@bmcfee, looks like the above Issue / PR has been closed.  Can we use `onset_detect` and `onset_backtrack` to generate segments?

**Comment by bmcfee** [comment]
Yup, it ought to work.

**Comment by tkell** [comment]
Closed with #95 !
